container/common/container_hints.go

```diff
@@ -25,7 +25,7 @@ import (
 	"os"
 )
 
-var argContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
+var ArgContainerHints = flag.String("container_hints", "/etc/cadvisor/container_hints.json", "location of the container hints file")
```

Comment:
This is not a test flag. It is being used AFAIK.
Also as part of this change, can we move this flag further up and pass it as an arg?